opencti-platform/opencti-graphql/src/migrations/1701354091174-files-registration.js

```diff
@@ -0,0 +1,23 @@
+import { executionContext, SYSTEM_USER } from '../utils/access';
+import { rawFilesListing } from '../database/file-storage';
+import { logApp } from '../config/conf';
+import { buildFileDataForIndexing } from '../modules/internal/document/document-domain';
+import { elIndexElements } from '../database/engine';
+import { INDEX_INTERNAL_OBJECTS } from '../database/utils';
+
+export const up = async (next) => {
+  const context = executionContext('migration');
+  logApp.info('[MIGRATION] Starting 1701354091161-files-registration.js');
+  const files = await rawFilesListing(context, SYSTEM_USER, '/', { recursive: true });
```

Comment:
Thats the initial loading for migration, do you have an alternative in mind?

Reply:
not yet

Reply:
The concept is to index all the files contains in minio, not just a subset.
In case of migration i  think its not really a problem if this listing take 10 or more secs